engines/wintermute/base/gfx/opengl/base_render_opengl3d.cpp

```diff
@@ -442,7 +442,7 @@ bool BaseRenderOpenGL3D::drawSpriteEx(BaseSurfaceOpenGL3D &tex, const Wintermute
 	vertices[2 * vertexSize + 8 + 2] = b;
 	vertices[2 * vertexSize + 8 + 3] = a;
 	vertices[3 * vertexSize + 8 + 0] = r;
-	vertices[3 * vertexSize + 8 + 2] = g;
+	vertices[3 * vertexSize + 8 + 1] = g;
```

Comment:
I'd suggest using a proper struct instead, or at the very least constants, to reduce the risk of such mistakes.

Reply:
Put in a struct for this